hknweb/events/templates/events/show_details.html

```diff
@@ -3,63 +3,101 @@
-
-  <p><b>Location</b>: {{ event.location }}</p>
-  
-  <div id="date-and-time"></div>
-  <!-- Script to render dates prettily -->
-  <script>
-    var start = new Date('{{ event.start_time|date:"c" }}');
-    var end = new Date('{{ event.end_time|date:"c" }}');
-    var sameDayEvent = start.getDate() === end.getDate() && start.getMonth() === end.getMonth() && start.getYear() === end.getYear();
-    
-    if (sameDayEvent) {
-      var HTMLtext = '<p><b>Date</b>: ' + start.toLocaleDateString() + 
-                     '</p> <p><b>Time</b>: ' + start.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'}) + ' to ' + 
-                     end.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'});
-    } else { // Event starts and ends on different days
-      var HTMLtext = '<p><b>Starts</b>: ' + start.toLocaleDateString() + ' at ' + 
-                     start.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'}) + '</p> <p><b>Ends</b>: ' + 
-                     end.toLocaleDateString() + ' at ' + end.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'});
+{% block header %}
+<link rel="stylesheet" href="{% static "events/style.css" %}">
+<style>
+    .heading {
+        display: none;
     }
-    document.getElementById("date-and-time").innerHTML = HTMLtext;
-  </script>
+    @media screen and (min-width: 1000px) {
+        .content {
+            padding-bottom: 1em;
+        }
+    }
+</style>
+{% endblock %}
+
+{% block content %}
+<div class="parent">
+  <h1 id="event-detail-title"> {{ event.name }} </h1>
+
+  <div id="left-details">
+
+    <p>{{ event.description }}</p>
+
+    <p><b>Event Type</b>: {{ event.event_type }}</p>
 
-  <p><a href="{% url 'events:index' %}">
-      Go Back to Calendar
-  </a></p>
+    <p><b>Location</b>: {{ event.location }}</p>
 
-  <br>
+    <div id="date-and-time"></div>
+    <!-- Script to render dates prettily -->
+    <script>
+      var start = new Date('{{ event.start_time|date:"c" }}');
+      var end = new Date('{{ event.end_time|date:"c" }}');
+      var sameDayEvent = start.getDate() === end.getDate() && start.getMonth() === end.getMonth() && start.getYear() === end.getYear();
+
+      if (sameDayEvent) {
+        var HTMLtext = '<p><b>Date</b>: ' + start.toLocaleDateString() +
+                       '</p> <p><b>Time</b>: ' + start.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'}) + ' to ' +
+                       end.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'});
+      } else { // Event starts and ends on different days
+        var HTMLtext = '<p><b>Starts</b>: ' + start.toLocaleDateString() + ' at ' +
+                       start.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'}) + '</p> <p><b>Ends</b>: ' +
+                       end.toLocaleDateString() + ' at ' + end.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'});
+      }
+      document.getElementById("date-and-time").innerHTML = HTMLtext;
```

Comment:
How do you think we should reconcile this with #130? Merge #130 first and / or merge its changes into this pull request?

Reply:
Probably merge #130 into this